logger/logger.go

```diff
@@ -23,27 +27,44 @@ type Logger interface {
 	Error(string)
 }
 
+var logLevels = map[string]level.Option{
+	"debug": level.AllowDebug(),
+	"info":  level.AllowInfo(),
+	"warn":  level.AllowWarn(),
+	"error": level.AllowError(),
+}
+
 var _ Logger = (*logger)(nil)
 
 type logger struct {
 	kitLogger log.Logger
 }
 
 // New returns wrapped go kit logger.
-func New(out io.Writer) Logger {
+func New(out io.Writer, logLevel string) Logger {
```

Comment:
Use this enumeration instead of string.

Reply:
Please take a look at my above comment. I have New accept a string here because that will be coming directly from env-vars like [here](https://github.com/mainflux/mainflux/blob/b07c6ae26499758753de2b223fab747610dceef9/cmd/http/main.go#L51-L55)

Reply:
You can use function similar to [this one](https://github.com/uber-go/zap/blob/f4243df2af592a2e0ccdccb77e1ad44f78d735e5/zapcore/level.go#L116) to unmarshal text into a level struct.